NEWS.md

```diff
@@ -11,6 +11,9 @@ New language features
 Language changes
 ----------------
 
+* Newly created Task objects (`@spawn`, `@async`, etc.) now adopt the world-age for methods from their parent
+  task, instead of using the global latest world. This is done to enable inference to eventually optimize
```

Comment:
Maybe makes sense to emphasise that the world is frozen at creation, and not schedule, time?

```suggestion
* Newly created `Task` objects (`@spawn`, `@async`, etc.) now adopt the world-age for
  methods from their parent task in which the `Task` objects are constructed, instead of
  using the global latest world. This is done to enable inference to eventually optimize
```

This is for clarifying that in

```julia
julia> runner = Channel() do runner
           foreach(schedule, runner)
       end

julia> # new world

julia> put!(runner, Task(f))
```

`f` is called with the world age of `Task(f)` not inside the `do` block.

Reply:
I don't think this clarifies it